src/java.base/share/classes/java/lang/foreign/FunctionDescriptor.java

```diff
@@ -49,21 +49,21 @@
 public sealed interface FunctionDescriptor permits FunctionDescriptorImpl {
 
     /**
-     * {@return the return layout (if any) associated with this function descriptor}
+     * {@return the return layout (if any) of this function descriptor}
      */
     Optional<MemoryLayout> returnLayout();
 
     /**
-     * {@return the argument layouts associated with this function descriptor (as an immutable list)}.
+     * {@return the argument layouts of this function descriptor (as an immutable list)}.
```

Comment:
I assume this should be "unmodifiable" rather than immutable here.

Reply:
This was derived from MethodType::parameterList, which also mentions immutable. I'm fine changing to "unmodifiable".